packages/apollo/package.json

```diff
@@ -38,6 +41,9 @@
     "apollo-server-express": "3.11.1",
     "apollo-server-fastify": "3.11.1",
     "apollo-server-plugin-response-cache": "3.8.1",
+    "body-parser": "^1.20.1",
+    "cors": "^2.8.5",
+    "express": "^4.18.2",
```

Comment:
I think all these should be peer dependencies base on what server your want to use (express or fastify)